CodeChat/CommentDelimiterInfo.py

```diff
@@ -86,16 +86,10 @@
   'SQL':            ( 2,      2,            2),
   ##                  #,     <#,            #>
   'PowerShell':     ( 1,      2,            2),
-
-  # These languages have failing unit tests
-  # ---------------------------------------
-  ##                 //,     /*,            */
-  'Sass':           ( 2,      2,            2),
-
-  # These langauges have **NOT** been tested.
-  # -----------------------------------------
+  # ``/*`` ~ ``*/`` tests fail.
   ##                  #,     /*,            */
   'GAS':            ( 1,      2,            2),
+  # ``;`` tests fail.
```

Comment:
Looking at the Pygments source, https://bitbucket.org/birkenfeld/pygments-main/src/e79a7126551c39d5f8c1b83a79c14e86992155a4/pygments/lexers/asm.py?at=default&fileviewer=file-view-default#asm.py-57 shows that it only recognizes single-line comments (`'#.*?$'` is a regular expression for a string starting with `#`, the followed by other characters (the `.*?`) until the end of the line (`$`). So, add a comment stating that since Pygments doesn't lex multi-line comments, we don't support them, and remove the tests from CodeToRest_test.py.